### ISSUE TITLE
"invalid digit found in string" spamming logs

### ISSUE THREAD

**Issue by sbach89** [SuperioOne/nut_webgui#1]
I'm using nut_webgui with docker compose on the same VM that nut is installed.

`docker-compose.yml`
```
services:
  nutweb:
    image: ghcr.io/superioone/nut_webgui:latest
    container_name: nutweb
    environment:
      - POLL_FREQ=60
      - POLL_INTERVAL=5
      - UPSD_ADDR=127.0.0.1
      - UPSD_USER=admin
      - UPSD_PASS=xxxxx
      - PORT=9000
    restart: unless-stopped
    network_mode: host
```
When it's running it's spamming the following error every 30 seconds.
```
ERROR nut_webgui::ups_service::ups_poll_service: Poll service error occurred: ClientError(ParseError("invalid digit found in string"))
```
as well as, but not as often:
```
WARN nut_webgui::ups_service::storage_service: Partial update ignored. Ups APC2 does not exists anymore.
```
I have 2 UPS's classified as 'APC1' and 'APC2'. The webui works, but only APC1 shows up, after about half an hour APC2 shows up, but the ERROR and WARN logs persist. Both UPS's show up with NUT's default CGI page as well as `upsc` command.

**Comment by SuperioOne** [comment]
Thanks for the issue report,

It appears that the issue is related to number parsing and how UPS devices reports their variables (expecting an integer as value , but getting a float instead). It also explains why APC2 doesn't show up properly.

I'll run some test on dummy-ups devices and rollout a fix soon as possible.

**Comment by sbach89** [comment]
Appreciate it, if you need any assistance with testing, let me know!

**Comment by SuperioOne** [comment]
Fix is live with v0.3.2.

If the issue still occurs, I can use output from `upsc <ups-name>` to simulate the UPS.

**Comment by sbach89** [comment]
Works great, thank you!
